meshnets/tune.py

```diff
@@ -92,53 +96,58 @@ def main(_):
         val_datasets.append(validation_dataset)
 
     # Define the search space over which `tune` will run.
-    config = {
+    search_space = {
         'batch_size':
             tune.grid_search(list(map(int, FLAGS.batch_size))),
         'latent_size':
             tune.grid_search(list(map(int, FLAGS.latent_size))),
         'num_mlp_layers':
             tune.grid_search(list(map(int, FLAGS.num_mlp_layers))),
-        'message_passing_steps':
-            tune.grid_search(list(map(int, FLAGS.message_passing_steps))),
         'learning_rate':
-            tune.grid_search(list(map(float, FLAGS.learning_rate)))
+            tune.grid_search(list(map(float, FLAGS.learning_rate))),
+        'message_passing_steps':
+            tune.grid_search(list(map(int, FLAGS.message_passing_steps)))
     }
 
-    experiment_config = {
-        'experiment_name': FLAGS.experiment_name,
+    config = {
         'num_workers_loader': FLAGS.num_workers_loader,
-        'tuning_run': TUNING_RUN,
-        # The two following parameters are unused in tuning regime
-        'num_workers_ray': None,
-        'num_cpus_per_worker': None,
-        'use_gpu': bool(FLAGS.num_gpus_per_worker),
+        'experiment_name': FLAGS.experiment_name,
         'max_epochs': FLAGS.max_epochs,
         'log_every_n_steps': FLAGS.log_every_n_steps,
         'save_top_k': FLAGS.save_top_k,
+        'train_dataset': train_dataset,
+        'validation_datasets_names': val_datasets_names,
+        'validation_datasets': val_datasets
     }
 
+    mlflow.create_experiment(FLAGS.experiment_name)
+
     # Alocate resources per trial.
-    resources_per_trial = {
+    resources_per_worker = {
```

Comment:
What is the difference? Can you update the comment if there is.

Reply:
There really is not much difference. But I will change the names accordingly.